swagger.yaml

```diff
@@ -55,6 +55,19 @@ definitions:
       - RWF
       - CNY
       - XAF
+      - ARS
```

Comment:
thanks, can you also add these to the FiatTypeEnum in the `fiatconnect-api.md` file?

Reply:
thanks for the review. Sure, already done